pkg/cmd/server/admin/create_signercert.go

```diff
@@ -99,9 +96,9 @@ func (o CreateSignerCertOptions) CreateSignerCert() (*crypto.CA, error) {
 		ca, written, err = crypto.EnsureCA(o.CertFile, o.KeyFile, o.SerialFile, o.Name)
 	}
 	if written {
-		fmt.Fprintf(o.Output.Get(), "Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
+		glog.V(3).Infof("Generated new CA for %s: cert in %s and key in %s\n", o.Name, o.CertFile, o.KeyFile)
```

Comment:
Write is unstructured.  Most cases are going to want structured.

----- Original Message -----
> > @@ -99,9 +96,9 @@ func (o CreateSignerCertOptions) CreateSignerCert()
> > (*crypto.CA, error) {
> >  		ca, written, err = crypto.EnsureCA(o.CertFile, o.KeyFile, o.SerialFile,
> >  		o.Name)
> >  	}
> >  	if written {
> > -		fmt.Fprintf(o.Output.Get(), "Generated new CA for %s: cert in %s and key
> > in %s\n", o.Name, o.CertFile, o.KeyFile)
> > +		glog.V(3).Infof("Generated new CA for %s: cert in %s and key in %s\n",
> > o.Name, o.CertFile, o.KeyFile)
> 
> You can already do that by passing in a writer with a Write "callback" of
> your design. It's kind of limited, but unless you want to do something more
> interesting than write (or not) the message, it seems adequate.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/2835/files#r31817606